draft-ietf-httpbis-retrofit.md

```diff
@@ -189,86 +208,91 @@ Date: Sun, 06 Nov 1994 08:49:37 GMT
 Its value would be mapped to:
 
 ~~~ http-message-new
-SF-Date: @784111777
+@784111777
 ~~~
 
-As in {{compatible}}, these fields are unable to carry values that are not valid Structured Fields, and so an application using this specification will need to how to support such values. Typically, handling them using the original field name is sufficient.
-
-Each field name listed below indicates a replacement field name and a means of mapping its original value into a Structured Field.
+Unlike those listed in {{compatible}}, these representations are not compatible with the original fields' syntax, and MUST NOT be used unless they are explicitly supported. For example, this means that sending them to a next-hop recipient in HTTP requires prior negotiation. This specification does not define how to do so.
```

Comment:
Is it worth adding, either here, or in security considerations a sentence like "As these representations are incompatible, any negotiation mechanism MUST unambiguously indicate which representation is in use at any time."

Reply:
Hmm, that's a bit weird as a requirement...

Reply:
Maybe there's a better way to phrase it. Intent is that it's not sufficient to just say "I support it", but you also need to design a way to indicate when you're using it. E.g. that binary sf flag, or in an API it may just come from you calling the sf-specific API, etc.

I suppose we can also leave that as an exercise to the reader, since it's kinda implied from them being different syntax.

Reply:
I've added 'and unambiguously'. This statement is made in several ways throughout the spec, I'm reluctant to effectively replicate it again...